src/shared/features/AppletSettings/LiveResponseStreamingSetting/LiveResponseStreamingSetting.tsx

```diff
@@ -1,28 +1,97 @@
-import { useFormContext } from 'react-hook-form';
-import { Trans, useTranslation } from 'react-i18next';
+import { ChangeEvent } from 'react';
+import { useWatch } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
 
 import { CheckboxController } from 'shared/components/FormComponents';
-import { StyledBodyLarge, theme } from 'shared/styles';
+import { StyledBodyLarge, theme, variables } from 'shared/styles';
+import { InputController } from 'shared/components/FormComponents';
+import { useCustomFormContext } from 'modules/Builder/hooks';
 
 import { StyledAppletSettingsDescription } from '../AppletSettings.styles';
+import { StyledController } from './LiveResponseStreamingSetting.styles';
+import { OnInputChange } from './LiveResponseStreamingSetting.types';
 
 export const LiveResponseStreamingSetting = () => {
   const { t } = useTranslation('app');
+  const { control, setValue } = useCustomFormContext();
+  const [streamEnabled] = useWatch({ name: ['streamEnabled'] });
+  const dataTestid = 'applet-settings-live-response-streaming';
+  const ipAddressFieldName = 'streamIpAddress';
+  const portFieldName = 'streamPort';
```

Comment:
Could you please move these variables outside of the component (no need initialize these variables each time when component is rendered)?